packages/react-magma-dom/src/components/InputBase/index.tsx

```diff
@@ -199,26 +199,28 @@ function getInputPadding(props: InputBaseStylesProps) {
         padding.left = props.theme.spaceScale.spacing10;
       }
     }
-  } else if (inputSize === 'medium') {
-    if (isClearable) {
-      if (iconPosition === 'right') {
-        padding.right = '68px';
-      } else if (iconPosition === 'left') {
-        padding.left = props.theme.spaceScale.spacing09;
-        padding.right = props.theme.spaceScale.spacing09;
+  } if (inputSize === 'medium') {
```

Comment:
This change is causing issues in the Icon Positions story: 
![image](https://user-images.githubusercontent.com/91160746/224102558-3c5d190f-5c8e-42fe-9be5-a02d7ad688d3.png)